src/designsystem/Button.tsx

```diff
@@ -1,52 +1,38 @@
 import React from 'react';
-import './button.css';
 
-interface ButtonProps {
-  /**
-   * Is this the principal call to action on the page?
-   */
-  primary?: boolean;
-  /**
-   * What background color to use
-   */
-  backgroundColor?: string;
-  /**
-   * How large should the button be?
-   */
-  size?: 'small' | 'medium' | 'large';
-  /**
-   * Button contents
-   */
-  label: string;
-  /**
-   * Optional click handler
-   */
-  onClick?: () => void;
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  isDisabled?: boolean;
```

Comment:
You don't need to use `isDisabled`. `HTMLButtonElement` has already its own `disabled` props. You can use that instead